src/index.html

```diff
@@ -11,7 +11,129 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset class="form">
+        <legend>Personal information</legend>
+        <label class="form-field">Surname:
+          <input
+            type="text"
+            name="Surname"
+            id="usersSurname"
+          >
+        </label>
+        <label class="form-field">Name:
+          <input
+            type="text"
+            name="usersName"
+          >
+        </label>
+        <label class="form-field">How old are You?
+          <input
+            type="number"
+            name="usersAge"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+          >
+        </label>
+        <label class="form-field">I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms&conditions"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form">
+        <legend>Registration</legend>
+        <label class="form-field">E-mail:
+          <input
+            type="email"
+            name="usersEmail"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-field">Password:
+          <input
+            type="password"
+            name="usersPassword"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">Do you love cats?
+          <input
+            type="radio"
+            name="Yes"
+          >Yes
+          <input
+            type="radio"
+            name="No"
+          >No
+        </label>
```

Comment:
Only one radio button can be chosen in one moment. Also, each input should have its own label but don't use a common label for them, because when you click on the question, yes becomes selected